ibtk/include/ibtk/FEProjector.h

```diff
@@ -65,6 +65,15 @@ class FEProjector
     std::pair<libMesh::PetscLinearSolver<double>*, libMesh::PetscMatrix<double>*>
     buildL2ProjectionSolver(const std::string& system_name);
 
+    /*!
+     * \return Pointers to a linear solver and sparse matrix corresponding to a
+     * L2 projection operator generated with the relevant nodal quadrature rule
+     * (i.e., QSIMPSON for second-order elements and QTRAP for first-order
```

Comment:
Does this comment need to be updated (since we are not currently using `QTRAP` or `QSIMPSON`)?

Reply:
You're right: let me add a citation of Hinton instead.

Reply:
I'm also OK to do something else

Reply:
...but maybe that should be another PR.

Reply:
Nah, I added it to this one. It only took a minute to fix the documentation :)